app/templates/_package.json

```diff
@@ -9,28 +9,28 @@
     "url": "<%= authorURL %>"
   },
   "scripts": {
+    "setup": "wp core download && npm install && bower install && npm run install-wp-plugins",
     "start": "grunt serve",
     "build": "grunt build",
-    "deploy": "git push && git push staging development",
+    "deploy": "git push origin development",
 
     "db-pull": "sh ./db/pull.sh",
     "db-push": "sh ./db/push.sh",
-    "install-plugins": "cd scripts && sh download-plugins.sh && cd ../"
+    "install-wp-plugins": "sh ./scripts/download-plugins.sh",
+    "install-db-sync": "cd ./db/ && git clone git@github.com:micjamking/wordpress-db-sync.git && cp -R wordpress-db-sync/* ./ && rm -rf wordpress-db-sync/"
   },
   "private": true,
   "devDependencies": {
```

Comment:
Maybe we should update the versions of the devDependencies to "*" so we always download the latest, rather than pinning the generator to a specific version.

Thoughts?